lib/browser/api/browser-window.js

```diff
@@ -80,11 +80,51 @@ BrowserWindow.prototype._init = function () {
     set: (autoHide) => this.setAutoHideMenuBar(autoHide)
   })
 
+  Object.defineProperty(this, 'visibleOnAllWorkspaces', {
+    get: () => this.isVisibleOnAllWorkspaces(),
+    set: (visible) => this.setVisibleOnAllWorkspaces(visible)
+  })
+
+  Object.defineProperty(this, 'simpleFullScreen', {
+    get: () => this.isSimpleFullScreen(),
+    set: (simple) => this.setSimpleFullScreen(simple)
+  })
+
+  Object.defineProperty(this, 'kiosk', {
+    get: () => this.isKiosk(),
+    set: (kiosk) => this.setKiosk(kiosk)
+  })
+
+  Object.defineProperty(this, 'documentEdited', {
+    get: () => this.isFullscreen(),
+    set: (edited) => this.setDocumentEdited(edited)
+  })
+
+  Object.defineProperty(this, 'fullscreen', {
+    get: () => this.isFullScreen(),
+    set: (full) => this.setFullScreen(full)
+  })
+
+  Object.defineProperty(this, 'shadow', {
```

Comment:
I think we should use `hasShadow` otherwise it would cause problems when writing documentations.

Reply:
The getter is already called `hasShadow`, so it'll error on duplicate identifiers if i do so